Library/Homebrew/extend/ARGV.rb

```diff
@@ -268,6 +268,10 @@ def fetch_head?
     include? "--fetch-HEAD"
   end
 
+  def ignore_installed_specs?
+    include? "--ignore-installed-specs"
```

Comment:
@vladshablinsky Sounds like to avoid breakage and prioritise safety: always build from source and don't allow an override to use the bottles (or just continue to ensure `--force-bottle` works) so we don't need an option.